backup/backup_client.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash                                                                                                                                                 
+
+set -euo pipefail
+
+if [ "${1:-}" = "backup-db" ] ; then
+    PGPASSWORD=${DB_PASS} pg_dump -h ${DB_HOST} -p ${DB_PORT} -U ${DB_USER}
+    exit
+fi
+
+caddy_paths="/opt/caddy"
+mastodon_paths="/opt/mastodon/public /opt/mastodon/elasticsearch /opt/mastodon/redis /opt/mastodon/mastodon.env /opt/mastodon/versions.env"
+monitoring_paths="/opt/mastodon/grafana/ /opt/mastodon/prometheus/ /opt/mastodon/monitoring-versions.env /opt/mastodon/monitoring-admin.env /opt/mastodon/postgres-monitoring-queries.yaml"
+extra_paths="/opt/backup /opt/mastodon/wait-for-container.sh /opt/mastodon/setup-mastodon.sh"
```

Comment:
I'd use arrays for those, like for the `remote` variable in the other script. But maybe in this case it's just busy work.